docs/source/integrations/building-integrations.md

```diff
@@ -196,10 +151,11 @@ interface HTTPGraphQLRequest {
 }
 ```
 
-We now have the `method`, `body`, and `search` properties computed. The only
-thing left for us to compute is the `headers` object! Apollo Server expects a
-`Map` of headers. In the Express implementation, we construct the `Map` by
-iterating over the `headers` object like so:
+Apollo Server handles the logic of extracting the HTTP method (i.e., `GET` vs. `POST`), relevant headers, and it knows whether to check the `body` or `search` fields for the GraphQL-specific parts of a query. So, Apollo Server handles your `method`, `body`, and `search` properties for the `HTTPGraphQLRequest`.
```

Comment:
Saying that Apollo Server extracts these is wrong, those bits are specifically up to the user to compute and hand off to Apollo Server. What happens after that is the "logic" mentioned above w.r.t. handling the request correctly based on the method provided, headers, etc.

Reply:
I replaced what was written here with what you originally had.